tests/deltatuner/finetune/merge_model/readme.md

```diff
@@ -28,7 +27,6 @@ python instruction_tuning_pipeline/finetune_clm.py \
         --output_dir "$DATA_PATH/llama2-7b-ssf-denas-bf16" \
         --delta ssf \
         --denas True \
-        --bf16 True \
```

Comment:
why remove bf16, it's a standard evaluation environment

Reply:
User can adjust it according to their server, I have add the instruction in the example readme file.